src/fable-library-ts/Util.ts

```diff
@@ -456,10 +458,16 @@ function equalObjects(x: { [k: string]: any }, y: { [k: string]: any }): boolean
   return true;
 }
 
-export function physicalEquality<T>(x: T, y: T): boolean {
+export function physicalEquals<T>(x: T, y: T): boolean {
   return x === y;
 }
```

Comment:
@ncave Is this rename expected?

Was the name of this function always wrong ?

Reply:
@MangelMaxime It's actually not used right now, it just sounds more better.